src/css/_utils.scss

```
@@ -30,3 +30,8 @@ $easeInOutSine: cubic-bezier(0.445, 0.05, 0.55, 0.95);
 $easeInOutExpo: cubic-bezier(1, 0, 0, 1);
 $easeInOutCirc: cubic-bezier(0.785, 0.135, 0.15, 0.86);
 $easeInOutBack: cubic-bezier(0.68, -0.55, 0.265, 1.55);
+
+// stylelint-disable declaration-no-important
+.bg-dark {
+  background-color: rgba(0, 0, 0, 0.7) !important;
```

It doesn't seem like `!important` is necessary here?

---

I added `!important` here because I consider this a utility where I expect it to always override the property.

---

```suggestion
  background-color: rgba(0, 0, 0, 0.7);
```

I'd rather avoid `!important` if possible. If we get to a situation where it's needed, we can tackle it then. But the cascade is working as expected right now.

---

Fine by me, I'm used to it from the Bootstrap utils :)